pkg/receive/request_limiter.go

```diff
@@ -19,14 +19,20 @@ var unlimitedRequestLimitsConfig = newEmptyRequestLimitsConfig().
 	SetSeriesLimit(0).
 	SetSamplesLimit(0)
 
+type requestLimiter interface {
+	AllowSizeBytes(tenant string, contentLengthBytes int64) bool
+	AllowSeries(tenant string, amount int64) bool
+	AllowSamples(tenant string, amount int64) bool
+}
+
```

Comment:
Why was this moved far away from the place where it's used?

Reply:
I thought it might be better to have interface and implementation in the same file, but you're right, it might make it more confusing. Let me also move the `headSeriesLimiter` interface as well. 🙂